### PR TITLE
return workflow result with container name

### PR DESCRIPTION
**Pull request by ajaxjiang96**

### CODE REVIEW

File: workflow.go [hunk 1]
```diff
@@ -68,7 +72,7 @@ func simulatorStartingWorkflow(ctx workflow.Context,
 	err = runDocker(ctx, strconv.Itoa(port), containerName)
 	if err != nil {
 		workflow.GetLogger(ctx).Error("Workflow failed.", zap.String("Error", err.Error()))
-		return err
+		return nil, err
```
Comment (qs-wang) [inline workflow.go]: return SimulatorStartResult{}, err

File: workflow.go [hunk 2]
```diff
@@ -48,12 +52,12 @@ func simulatorStartingWorkflow(ctx workflow.Context,
...
 	port, err := freeport.GetFreePort()
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	sid, err := shortid.New(1, shortid.DefaultABC, uint64(time.Now().UnixNano()))
 	if err != nil {
-		return err
+		return nil, err
```
Comment (qs-wang) [inline workflow.go]: return SimulatorStartResult{}, err

File: workflow.go [hunk 3]
```diff
@@ -48,12 +52,12 @@ func simulatorStartingWorkflow(ctx workflow.Context,
 
 	port, err := freeport.GetFreePort()
 	if err != nil {
-		return err
+		return nil, err
```
Comment (qs-wang) [inline workflow.go]: return SimulatorStartResult{}, err

**Review by qs-wang — CHANGES_REQUESTED** [review]

File: workflow.go (hunk 3 above)
Comment (ajaxjiang96) [reply workflow.go]: Done

File: workflow.go (hunk 1 above)
Comment (ajaxjiang96) [reply workflow.go]: Done

**Review by qs-wang — APPROVED** [review]

**Pull request merged**